docs/verify/index.mdx

```diff
@@ -17,7 +17,17 @@ Stabilizer circuits, also known as Clifford circuits, are a useful tool for acco
 
 For general quantum circuits, the following tools are available to verify your quantum programs:
 
-- For simulating circuits when using Qiskit Runtime, use its [local testing mode.](local-testing-mode) 
+- For simulating circuits when using Qiskit Runtime, use its [local testing mode.](local-testing-mode)
 - For exact simulation of small quantum circuits, you can use the reference primitives included with Qiskit. See [Exact simulation with Qiskit primitives](simulate-with-qiskit-primitives).
 - For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer) (V1 primitive support only). For V2 primitive support, use the [Qiskit Runtime local testing mode](local-testing-mode) with a Qiskit Aer backend.
-- To build custom noise models, use the [`noise`](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html) module of Qiskit Aer. See [Building noise models](building_noise_models).
\ No newline at end of file
+- To build custom noise models, use the [`noise`](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html) module of Qiskit Aer. See [Building noise models](building_noise_models).
+
+## Hardware considerations
+
+Several factors impact memory used in quantum simulation, so there are no exact hardware requirements for simulation, but there are some guidelines you can follow.
+
+- The only requrement for running Qiskit is that there is a functioning Python environment installed.  For details, see the [Operating system support](../start/install#operating-system-support) section.
+- Because the requirements for simulating quantum circuits scales exponentially with the number of qubits, the available hardware limits the number of qubits that can be simulated. For example, a system with 4GB RAM can simulate approximately 27 qubits.
```

Comment:
```suggestion
- Because the requirements for simulating quantum circuits scale exponentially with the number of qubits, the available hardware limits the number of qubits that can be simulated. For example, a system with 4GB RAM can simulate approximately 27 qubits.
```